src/common/models/xuiNode.class.ts

```diff
@@ -38,7 +38,6 @@ export class XuiNode extends EventEmitter {
     public authenticateDefault = (req: Request, res: Response, next: NextFunction): void => {
         if (req.isUnauthenticated()) {
             this.logger.log('unauthenticated, redirecting')
```

Comment:
And this log message shouldn't say it's redirecting when it's not. 😁

Reply:
same comment

Reply:
The log message is inaccurate and should be changed to reflect that it's no longer redirecting.

Reply:
Is it ok if I keep 'unauthenticated', and remove redirecting?

Reply:
Yes, absolutely! 😁

Reply:
updated as mentioned